cmd/stack_package.go

```diff
@@ -88,9 +92,134 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 			Info.Log("Running appsody stack package")
 			Info.Log("******************************************")
 
-			stackPath := rootConfig.ProjectDir
+			// get current time
+			currentTime := time.Now().Format(time.RFC3339)
+
+			projectPath := rootConfig.ProjectDir
+
+			// make a copy of the folder to apply template to
+			err := copyDir(projectPath, projectPath+"packagecopy")
```

Comment:
Had you considered putting the templating code in one or more functions? This function is very big already, I'd like to see it broken down.

Reply:
Thanks for doing this!

Reply:
Thanks for splitting up the functions! I think it looks much better :)